docs/SUMMARY.md

```diff
@@ -0,0 +1,60 @@
+# Table of contents
```

Comment:
This appears to be a machine-generated file, and it's already out of date. If we're going to be procedurally generating content, it should be at build/deploy time so it doesn't have to be separately maintained, and the code to do so should be clearly included in the repo or part of vanilla Redocly, not some unstated external script.